addons/xeh/xeh_a3/config.cpp

```diff
@@ -1,5 +1,5 @@
 class CfgPatches {
```

Comment:
```suggestion
#include "\x\cba\addons\xeh\xeh_a3\script_component.hpp"
class CfgPatches {
```

Reply:
I did actually add the file, just forgot to include it... Nice catch.